FEEDBACK.md

```diff
@@ -0,0 +1,23 @@
+## I would like feedback on the following:
+
+* Are my classes coupled enough?
+
+* Are they loosely coupled?
+
+* Could I have further broken down my classes? Do they have a single responsibility? I feel they might be getting a bit large!
+
+* How is my mocking? 
+
+* Could I have used keyword arguments better to make my code more changeable later?
+
+* How would these classes work together as a program if they are so loosely coupled. Do I need a customer ordering interface to implement the public methods of my classes? Or should one of my classes be doing this already e.g. restaurant?
+
+* Am I accessinging my attributes correctly in my restaurant methods? Should I be using the method rather than the instance variable?
+
+* I initialise my restaurant class with a pre-set menu. Is there a better way of doing this, e.g. with optional arguments or keyword arguments?
```

Comment:
I think it's fine! You don't need to overdo it: we want to keep our implementations as simple and close to the user stories as possible.